imaplugin/src/main/java/com/kaltura/playkit/plugins/ima/IMAConfig.java

```diff
@@ -31,6 +32,7 @@
     public static final int DEFAULT_AD_LOAD_TIMEOUT = 8;
     public static final int DEFAULT_CUE_POINTS_CHANGED_DELAY = 2000;
     public static final int DEFAULT_AD_LOAD_COUNT_DOWN_TICK = 250;
+    public static final String KALTURA_VP_ANDROID = "kaltura-vp-android";
```

Comment:
Rename to `AD_PLAYER_TYPE` (after renaming the current AD_PLAYER_TYPE, see below).
Add `AD_PLAYER_VERSION = VERSION_STRING`.

Then just use these constants directly without members+getters+setters. They will never change (in runtime) and nobody has to set them from outside.

Reply:
@noamtamim 
pushed